src/components/ContactCard/ContactFieldInput.jsx

```diff
@@ -42,6 +43,7 @@ const ContactFieldInput = ({
     <div className="contact-form-field__wrapper u-flex u-flex-column-s">
       <Field
         {...propsUpdated}
+        id={uniqueId('field_')}
```

Comment:
My understanding is that this will call `uniqueId()` on each `<ContactFieldInput>` rerender and so `id` will change too often

Suggestion: you can move it into a state like this and so id won't change during the component's life:
```jsx
const [id] = useState(uniqueId('field_'))

// ...

<Field id={id} ... />
```

Reply:
bien vu merci, je corriger avant de merger 👍